### PR TITLE
Remove redundant github.com/pkg/errors pkg

### PR DESCRIPTION
**Pull request by mresvanis**
This PR removes the `github.com/pkg/errors` package in favor of the standard [errors](https://pkg.go.dev/errors) package both in LCA and in imager. 

> We can wrap any error using `fmt.Errorf("... %w ...", err)` instead of the previous `github.com/pkg/errors` API `errors.Wrap(err, ...)` or `errors.Wrapf(err, ...)`.

### CODE REVIEW

File: ibu-imager/internal/ops/ops.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func (o *ops) SystemctlAction(action string, args ...string) (string, error) {
 	o.log.Infof("Running systemctl %s %s", action, args)
 	output, err := o.RunInHostNamespace("systemctl", append([]string{action}, args...)...)
 	if err != nil {
-		err = errors.Wrapf(err, "Failed executing systemctl %s %s", action, args)
+		err = fmt.Errorf("Failed executing systemctl %s %s: %w", action, args, err)
```
Comment (omertuc) [inline ibu-imager/internal/ops/ops.go:33]:
```suggestion
		err = fmt.Errorf("failed executing systemctl %s %s: %w", action, args, err)
```

Comment (mresvanis) [reply ibu-imager/internal/ops/ops.go]: Nice catch, updated :+1:

File: ibu-imager/internal/ops/execute.go [hunk 2]
```diff
@@ -33,5 +33,5 @@ func (e *executor) Execute(command string, args ...string) (string, error) {
 	cmd.Stdout = &stdoutBytes
 	cmd.Stderr = &stderrBytes
 	err := cmd.Run()
-	return strings.TrimSpace(stdoutBytes.String()), errors.Wrap(err, stderrBytes.String())
```
Comment (omertuc) [inline ibu-imager/internal/ops/execute.go:36]: Unlike `errors.Wrap`, `fmt.Errorf` will not return `nil` if the only instance of `%w` in its string is nil

File: ibu-imager/internal/ops/execute.go [hunk 3]
```diff
@@ -33,5 +33,5 @@ func (e *executor) Execute(command string, args ...string) (string, error) {
...
+	return strings.TrimSpace(stdoutBytes.String()), fmt.Errorf("%s: %w", stderrBytes.String(), err)
```
Comment (omertuc) [inline ibu-imager/internal/ops/execute.go:36]:
```suggestion
    stdoutBytesTrimmed = strings.TrimSpace(stdoutBytes.String())
    stderrBytesTrimmed = strings.TrimSpace(stderrBytes.String())
    if err != nil {
        return stdoutBytesTrimmed, fmt.Errorf("%s: %w", stderrBytes.String(), err)
    }

    return stdoutBytesTrimmed, nil
```

Comment (omertuc) [reply ibu-imager/internal/ops/execute.go:36]: (see [comment](https://github.com/openshift-kni/lifecycle-agent/pull/38/files#r1377851143))

Comment (mresvanis) [reply ibu-imager/internal/ops/execute.go]: Absolutely, great catch! Updated.

**Pull request merged**